vm/src/readline.rs

```diff
@@ -19,13 +17,16 @@ pub enum ReadlineResult {
 mod basic_readline {
     use super::*;
 
-    pub struct BasicReadline<'vm> {
-        vm: &'vm VirtualMachine,
+    pub trait Helper {}
```

Comment:
Maybe some more specific name then `Helper`?

Reply:
This trait plays a similar role to `rustyline::Helper`, and would be referenced as `readline::Helper`, which I think is pretty specific. Something more specific might be `ReadlineHelper`, but that is kinda redundant because of the `readline::` module prefix.